src/applications/edu-benefits/0993/helpers.js

```diff
@@ -24,3 +25,14 @@ export function transform(formConfig, form) {
     }
   });
 }
+/* eslint-disable */
+export const tabbableTitle = title => (<div
```

Comment:
Are there other candidates for using this?

Reply:
I can't think of any. Our pure content pages are pretty well separated from our field pages. This was a rare case where we wanted a disclosure without an acknowledgment like a checkbox. 

I do think we should probably come up with a better name and make it available as a new form type though- `tabbable` is babble